tests/migration_test.go

```diff
@@ -2248,18 +2249,33 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse(fedoraVMSize)
 				defer deleteDataVolume(dv)
 
-				By("Starting the VirtualMachineInstance")
-				vmi = runVMIAndExpectLaunch(vmi, 240)
-
-				By("Checking that the VirtualMachineInstance console has expected output")
-				Expect(libnet.WithIPv6(console.LoginToFedora)(vmi)).To(Succeed())
+				By("Limiting the bandwidth of migrations in the test namespace")
+				quantity := resource.MustParse("1Mi")
+				migrationPolicy := &v1alpha1.MigrationPolicy{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: util.NamespaceTestDefault,
+						Labels: map[string]string{
+							cleanup.TestLabelForNamespace(util.NamespaceTestDefault): "",
+						},
+					},
+					Spec: v1alpha1.MigrationPolicySpec{
+						BandwidthPerMigration: &quantity,
+						Selectors: &v1alpha1.Selectors{
+							NamespaceSelector: &metav1.LabelSelector{
```

Comment:
This will affect all vms in the namespace. Wouldn't it be better to use vmi selector? (for parallel run)

Reply:
Every test runs in its own namespace when executed in parallel. NamespaceTestDefault basically has a different name on each ginkgo execution node.

Reply:
:+1:  Great that we have migration policies :)

Reply:
Also I should start reading descriptions :)

Reply:
Do we clean `MigrationPolicy`?

Reply:
Yes, all MigrationPolicies which have the label

```
						Labels: map[string]string{
							cleanup.TestLabelForNamespace(util.NamespaceTestDefault): "",
						},
```

are cleaned when a test in the `NamespaceTestDefault` namespace is done (so only these to not influence each other).